packages/@lwc/engine-core/src/framework/base-bridge-element.ts

```diff
@@ -125,6 +132,43 @@ export function HTMLBridgeElementFactory(
         };
     }
     defineProperties(HTMLBridgeElement.prototype, descriptors);
+    // observable attributes
+    HTMLBridgeElement.prototype.attributeChangedCallback = function (
+        this: HTMLElement,
+        attrName: string,
+        oldValue: string,
+        newValue: string
+    ) {
+        if (oldValue === newValue) {
+            // Ignore same values.
+            return;
+        }
+        const propName = attributeToPropMap[attrName];
+        if (isUndefined(propName)) {
+            if (!isUndefined(superAttributeChangedCallback)) {
+                // delegate unknown attributes to the super.
+                superAttributeChangedCallback.apply(this, arguments);
```

Comment:
delegating to the super is important, in case the super defines other props that we don;'t know how to handle

Reply:
We should probably add a test for this.

Reply:
we have tons of tests for this... the base bridge is extended by any custom element... customization of subclasses is also in place today. what exactly do you want to test other than that?